src/cljs/nnts2/note/components.cljs

```diff
@@ -0,0 +1,47 @@
+(ns nnts2.note.components
+  (:require [re-frame.core :as re-frame]
+            [reagent.core :as r]))
+
+(enable-console-print!)
+
+(def full (r/atom ""))
+
+(defn write-notes []
+  (let [content-text (r/atom "")
+        title-text (r/atom "")]
```

Comment:
Maybe combine these and make an atomic map instead? Something like this:
````
[note (r/atom {:content ""
               :title   ""}]
````

Reply:
This becomes a little complicated if you are trying to clear the form. I cant use what you have suggested in that case